src/components/BorderedList/BorderedListItem/BorderedListItem.tsx

```diff
@@ -0,0 +1,31 @@
+import type { ListItemProps } from '@digdir/design-system-react';
+import { List } from '@digdir/design-system-react';
+import cn from 'classnames';
+import * as React from 'react';
+
+import classes from './BorderedListItem.module.css';
+
+export type BorderedListItemProps = {
+  /**
+   * Style of the border separating the items
+   */
+  borderStyle?: 'solid' | 'dashed';
+} & ListItemProps;
+
+export const BorderedListItem = ({
+  borderStyle = 'dashed',
+  children,
+  ...rest
+}: BorderedListItemProps) => {
+  return (
+    <List.Item
+      className={cn(classes.borderedListItem, { [classes[borderStyle]]: borderStyle })}
+      style={{ listStyle: 'none', marginBottom: 0 }}
```

Comment:
I don't see why we should have classes and set style prop if we can do it all with one of those. Can't you set these styles in css classes as well?